src/Overlay.tsx

```diff
@@ -10,6 +10,7 @@ import {useCombinedRefs} from './hooks/useCombinedRefs'
 type StyledOverlayProps = {
   width?: keyof typeof widthMap
   height?: keyof typeof heightMap
+  maxHeight?: keyof typeof heightMap
```

Comment:
Should we will want to exclude `auto` and `initial` on this one?

Reply:
Excluded in https://github.com/primer/components/pull/1287/commits/010c82ef6e38c63b2406e05079ac1b9ba3860cec